### PR TITLE
Completed some preliminary actions for diners and owners. Working on …

### PR DESCRIPTION
**Pull request by PLisak777**
Created Actions for both diner and owner. Will need to meetup with backend to implement some more endpoints to get everything working. Made a few changes to state to reflect some of the things we plan to do. Working on getting reducers going.

### CODE REVIEW

**Review by allyjay317 — COMMENTED** [review]
Don't see anything that should be a problem, some of the endpoint names might be a bit different, but that's a task for tomorrow after I further test the database I set up

File: food-truck-tracker/src/index.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+
 import './index.css';
 import App from './App';
-import { BrowserRouter as Router } from 'react-router-dom'
 
+const store = createStore(rootReducer, applyMiddleware(thunk))
 
 ReactDOM.render(
-  <Router>
+  <Provider store={store}>
```
Comment (ChadDiaz) [inline food-truck-tracker/src/index.js:17]: Would you put Router inside  your Provider before merging?  Thanks!

**Review by ChadDiaz — CHANGES_REQUESTED** [review]
Could you let me know why Router was changed in the index.js file?  Thanks.

**Comment by allyjay317** [comment]
Provider is needed for Redux you should be able to add the router back in as well though, inside the provider tag

**Pull request merged**